server/views/pages/create/caseload.njk

```diff
@@ -10,9 +10,9 @@
 {% set backLinkHref = "/" %}
 {% set noJsBackLink = true %}
 
-{% macro offenderName(offenderName, crn, prisonerNumber, probationPractitioner, index) %}
+{% macro offenderName(offenderName, crn, prisonerNumber, probationPractitioner, insidePilot, index) %}
     <div class="caseload-offender-name">
-        {% if probationPractitioner %}
+        {% if probationPractitioner and insidePilot %}
             <button id="name-button-{{ index }}" type="submit" name="prisonerNumber" value='{{ prisonerNumber }}' class="link-button govuk-heading-s govuk-!-padding-left-0 govuk-!-margin-bottom-0">{{ offenderName }}</button>
```

Comment:
Only show a create link where a probation practitioner exists AND the offender is inside a pilot establishment.